src/ILVerify/src/ILImporter.Verify.cs

```diff
@@ -1208,7 +1218,25 @@ void ImportStoreIndirect(TypeDesc type)
 
         void ImportThrow()
         {
-            throw new NotImplementedException();
+            var value = Pop();
+
+            var obj = _typeSystemContext.GetWellKnownType(WellKnownType.Object);
+            var exception = _typeSystemContext.GetWellKnownType(WellKnownType.Exception);
```

Comment:
Ok, I changed this. We discussed this and our approach now: By default we only check for CLI compliance, but we also plan to add a parameter to check for CLS compliance. But atm we ignore CLS.